test/integration/setup.go

```diff
@@ -166,15 +170,22 @@ func createBranchTable(ctx context.Context) (*bigtable.Table, error) {
 	return server.SetupBigtable(ctx, data)
 }
 
-func updateGolden(v interface{}, fname string) {
+func updateGolden(v interface{}, fname string, shard ...bool) {
 	jsonByte, _ := json.MarshalIndent(v, "", "  ")
-	err := ioutil.WriteFile(fname, jsonByte, 0644)
+	var err error
+	if len(shard) == 1 && shard[0] {
```

Comment:
So, is this the way to do default arg in golang?

Reply:
Yea, it's more of optional args and used as default arg, as there is no native default arg